### ISSUE TITLE
Aion Solo Pool error

### ISSUE THREAD

**Issue by CerealKillah** [aionnetwork/aion_miner#29]
While running Aion kernel 0.1.19 (any release) the Aion Solo Pool throws error listed below.

This only occurs in above version. When running 0.1.18 I don't get any errors and I can connect to the pool with the CUDA miner just fine.

`3863:C 10 Apr 18:41:44.281 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
3863:C 10 Apr 18:41:44.281 # Redis version=4.0.8, bits=64, commit=00000000, modified=0, pid=3863, just started
3863:C 10 Apr 18:41:44.281 # Configuration loaded
function verify() { [native code] }
2018-04-10 18:41:44 [POSIX]	[Connection Limit] (Safe to ignore) POSIX module not installed and resource (connection) limit was not raised
2018-04-10 18:41:44 [Master]	[CLI] CLI listening on port 17117
2018-04-10 18:41:44 [Master]	[PoolSpawner] Spawned 1 pool(s) on 1 thread(s)
function verify() { [native code] }
2018-04-10 18:41:45 [Switching]	[Setup] (Thread 1) Loading last proxy state from redis
2018-04-10 18:41:45 [Pool]	[aion] (Thread 1) Share processing setup with redis (127.0.0.1:6379)
2018-04-10 18:41:45 [Switching]	[Setup] (Thread 1) Switching "switch0" listening for equihash on port 3333 into aion
2018-04-10 18:41:45 [Pool]	[aion] (Thread 1) Could not start pool, error with init RPC validateaddress - {"code":-32601,"message":"Method not found"}
`

**Comment by CerealKillah** [comment]
I should add that I am running the 0.1.11 solo pool.

**Comment by CerealKillah** [comment]
Ofcourse, I've attached it here. (renamed to .txt for compatibility)

If it's a configuration mistake on my behalf, I do apologize.

[config.txt](https://github.com/aionnetwork/aion_miner/files/1895684/config.txt)

**Comment by aion-Ross** [comment]
Could you try changing line 10 to:

``` <apis-enabled>web3,eth,personal,stratum</apis-enabled> ```

Seems like the stratum API may not be enabled on the RPC server in the kernel.

**Comment by CerealKillah** [comment]
That did the trick. Sorry for spending your time.

May I suggest that as a default configuration? I noticed another person on your forums with the same issue a couple of days ago.

**Comment by aion-Ross** [comment]
Yep, I think we'll definitely look into either making it a default or make the required change more prominent in future releases.
